integration_tests/macros/create_old_audit_table.sql

```diff
@@ -4,13 +4,19 @@
 
     {{ logging.create_audit_schema() }}
 
+    {% if adapter.type() == 'bigquery' %}
+        {{ set string_type = 'string' }}
+    {% else %}
+        {{ set string_type = 'varchar(512)' }}
+    {% endif %}
+
     create table if not exists {{ logging.get_audit_relation() }}
     (
-       event_name       varchar(512),
-       event_timestamp  {{dbt_utils.type_timestamp()}},
-       event_schema     varchar(512),
-       event_model      varchar(512),
-       invocation_id    varchar(512)
+       event_name       {{ string_type }},
```

Comment:
Since `dbt_utils` is already a dependency for this package, I figure this should be
```suggestion
       event_name       {{ dbt_utils.type_string() }},
```
and then you can skip the logic above

Reply:
I was thinking of that as well but wasn't sure if it would break anything on the non-BigQuery side as `dbt_utils.type_string()` would evaluate to `varchar` instead of `varchar(512)` for other DBs.

Just want to confirm that you still think that's an okay change to make?

Reply:
Yes! Redshift, Postgres, and Snowflake are all okay with setting column types to `varchar` with no length specification. And if they did need a specification, that should be a matter for the `dbt_utils.type_string()` definition IMO